app/modules/set-selector/artists.factory.js

```diff
@@ -8,13 +8,8 @@ require('./set-selector.module.js')
 // @ngInject
 function ArtistsFactory($q, ArtistListConstants, localStorageService, zooAPIConfig, zooAPI) {
```

Comment:
`localStorageService` is no longer used by ArtistsFactory; can it be removed, or is it still required implicitly by `Angular.factory()`/etc?